From 8bbfb03badf9910f3e196a5055e7c77353dca280 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 17 Jul 2018 12:08:25 -0500
Subject: [PATCH] Update CLI internal version to match what source-build
 produces.

See https://github.com/dotnet/source-build/issues/657 for details.  This
patch can be removed after that issue is fixed.
---
 build/DependencyVersions.props | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/build/DependencyVersions.props b/build/DependencyVersions.props
index c80f2a630..15f27e97d 100644
--- a/build/DependencyVersions.props
+++ b/build/DependencyVersions.props
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <DotNetCoreSdkLKGVersion>3.0.100-preview1-009020</DotNetCoreSdkLKGVersion>
-    <DotnetCliInternalVersion>3.0.100-alpha1-20180720-2</DotnetCliInternalVersion>
+    <DotnetCliInternalVersion>3.0.100-alpha1-00000001-01</DotnetCliInternalVersion>
     <MicrosoftAspNetCoreAllPackageVersion>2.2.0-preview1-34355</MicrosoftAspNetCoreAllPackageVersion>
     <MicrosoftAspNetCoreAppPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreAppPackageVersion>
     <MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>2.1.1</MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>
-- 
2.18.0

